src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -239,6 +254,21 @@ IS 'directly sourced from moj_annotation_s';
 COMMENT ON COLUMN annotation.version_label
 IS 'inherited from dm_sysobject_r, for r_object_type of moj_annotation';
 
+CREATE TABLE annotation_document

```

Comment:
Do we know what columns should be defined as `NOT NULL`?

Reply:
all columns in annotation_document and transcription_document have been made not null as of v52 of schema script.